SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/ButtonNode.swift

```diff
@@ -41,36 +32,54 @@ class ButtonNode: Node {
         self.cardData = cardData
         
         self.button.setTitle(cardData.text, for: .normal)
+        self.button.titleLabel!.numberOfLines = 0; // Dynamic number of lines
+        self.button.titleLabel!.lineBreakMode = .byWordWrapping;
+        
+        let fontSize:CGFloat = GlobalSettings.getUserSettings().getFontSize()
+        let sizedFont:UIFont = self.button.titleLabel!.font!.withSize(fontSize)
+        self.button.titleLabel!.font = sizedFont
+        
+        if let cardImage:UIImage = UIImage(data:cardData.imagefile) {
+            let buttonWidth:CGFloat = self.button.frame.width
+            let buttonHeight:CGFloat = self.button.frame.height
+            let buttonAR:CGFloat = buttonWidth / buttonHeight // Button Aspect Ratio
+            let _buttonAR:CGFloat = 1.0 / buttonAR
+            
+            let imageBorder:CGFloat = fontSize
+            let imageTop:CGFloat = 12 * _buttonAR
+            let imageBottom:CGFloat = (2 * imageBorder - imageTop) * _buttonAR // Think about it...
```

Comment:
If I understand it correctly, in line number 49 you decide on the padding at the top of the image. Should `_buttonAR` be multiplied with `imageTop` again in line 50 when determining the padding at the cotton of the image?